app/services/stage_blobs.rb

```diff
@@ -1,39 +1,24 @@
 # frozen_string_literal: true
 
 # Moves files from the ActiveStorage to the staging mount
-class StageFiles
+class StageBlobs
   def self.stage(signed_ids, druid)
     # Skip side effects if no signed IDs provided
     return yield if signed_ids.empty?
 
-    @dir = StagingDirectory.new(druid: druid, staging_location: Settings.staging_location)
-
     blobs = Blobs.blobs_for(signed_ids)
-    if blobs.empty?
-      copy_globus(signed_ids)
-    else
-      copy_blobs(blobs)
-    end
-    yield
-  end
+    yield if blobs.empty?
 
-  def self.copy_blobs(blobs)
-    copy_files_to_staging(blobs)
+    copy_files_to_staging(druid, blobs)
+    yield
```

Comment:
There are now a few potential `yield`s in close proximity to one another (lines 7, 10, 13). Do we think only one of these will be triggered or is it intentional to have this method `yield` 2-3 times?

Reply:
Good catch, fixed.